packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -332,32 +333,16 @@ the body are the following properties if specified:
 }
 
 /**
- * A hook for performing actions with the Shopper Baskets API.
+ * A hook for performing mutations with the Shopper Baskets API.
  */
-// TODO: Why does prettier not like "extends `${Actions}`"?
 // eslint-disable-next-line prettier/prettier
-export function useShopperBasketsAction<Action extends `${ShopperBasketsActions}`>(
+export function useShopperBasketsMutation<Action extends `${ShopperBasketsActions}`>(
```

Comment:
Would this looks better if we declare the values of ShopperBasketActions as such? Prettier probably won't cut off the extends type with this
```
type Values = typeof ShopperBasketsActions[keyof typeof ShopperBasketsActions]
export function useShopperBasketsMutation<Action extends Values>
```